helm-charts/inbucket/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: Disposable webmail server (similar to Mailinator) with built in SMT
 name: inbucket
 type: application
 appVersion: 3.0.0
-version: 2.2.2
+version: 2.3.0
```

Comment:
sorry, this should be a patch version not a minor

Reply:
Fixed :)